cli/cli.ts

```diff
@@ -252,6 +252,14 @@ class FileGithubDb implements pxt.github.IGithubDb {
     loadPackageAsync(repopath: string, tag: string): Promise<pxt.github.CachedPackage> {
         return this.loadAsync(repopath, tag, "pkg", (r, t) => this.db.loadPackageAsync(r, t));
     }
+
+    loadTutorialMarkdown(repopath: string, tag?: string): Promise<pxt.github.CachedPackage> {
```

Comment:
Just curious why we need a function like this defined in so many places? I see that the one defined in the github namespace is getting used along with db.loadTutorialMarkdown, but what do each of these allow for us to do?

Reply:
Same question for the `cacheReposAsync` function.

Reply:
it's defined on the githubdb interface so all of the classes that implement that interface need to have it as well! i just propagated it to all of the existing ones we have, i don't actually know what this one in the CLI is used for.